src/main/java/com/epam/ta/reportportal/commons/querygen/CriteriaHolder.java

```diff
@@ -236,6 +241,29 @@ public Object castValue(String oneValue, ErrorType errorType) {
     return castedValue;
   }
 
+  private LocalDateTime parseDateTimeWithOffset(String timestamp) {
```

Comment:
Can we move it somewhere in different class/service/utils for further reusability?

Reply:
done